Library/Homebrew/utils/github.rb

```diff
@@ -593,18 +593,21 @@ def self.get_pull_request_changed_files(tap_remote_repo, pull_request)
     API.open_rest(url_to("repos", tap_remote_repo, "pulls", pull_request, "files"))
   end
 
+  private_class_method def self.authenticated_url!(url)
```

Comment:
```suggestion
  private_class_method def self.authenticate_url!(url)
```

Maybe?

Reply:
I agree it should probably not be past-tense, except that it makes it sound like it's performing some authentication request.

Reply:
Then something like `add_auth_token_to_url!` would probably be clearer.